app/Http/Controllers/AiConvController.php

```diff
@@ -246,12 +273,31 @@ public function updateMessage(Request $request, $slug) {
             'iv' => $validatedData['iv'],
             'tag' => $validatedData['tag'],
             'model' => $validatedData['model'],
-            'completion' => $validatedData['completion']
+            'completion' => $validatedData['completion'],
         ]);
 
+        $auxiliaries = [];
+        if (isset($validatedData['auxiliaries'])) {
+            // drop previous auxiliaries and recreate them
```

Comment:
There's no point in updating these -> they are specific to the individual chat message, and thus should be recreated.